CHANGELOG.md

```diff
@@ -1,79 +1,85 @@
-# v2.5.0
+# Change Log
+
+All notable changes to this project will be documented in this file
+automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## v2.5.0 - 2017-03-24
 
 * Forward options object to docker pull
 
-# v2.4.4
+## v2.4.4
```

Comment:
Well I *could've* also gone and added the dates for each release, but ain't nobody got time for that :P